docs/index.md

```diff
@@ -8,11 +8,14 @@ Here are the results from the last run associated with
   ![briowu-rho](img/brio-wu_test_rho.png){: style="width:400px"}
   ![briowu-bd2](img/brio-wu_test_bd2.png){: style="width:400px"}
   ![briowu-ud](img/brio-wu_test_ud.png){: style="width:400px"}
-
+  
+  **[Tables](tables/brio-wu.md)**
```

Comment:
Instead of naming the link Tables, it would be nice if we called it Comparison to reference